docs/resources/client/guideline-string.md

```diff
@@ -0,0 +1,40 @@
+# Client Guideline String Resource
+
+## Guideline String
+The guideline string is the linearized string format of the guidelines of a level. It is contained within the [inner level string](). It is formatted as follows:
```

Comment:
Please edit all the links that you have and provide them with actual redirects (if any).